src/Sulu/Bundle/TestBundle/Resources/app/config/sulu.yml

```diff
@@ -13,25 +13,6 @@ monolog:
         main:
             type:  'null'
 
-sulu_core:
-    cache_dir: '%kernel.root_dir%/cache/sulu'
-    webspace:
-        config_dir: "%kernel.root_dir%/Resources/webspaces"
-    content:
-        structure:
-             default_type:
-                snippet: 'default'
-             paths:
-                 pages:
-                     path: '%kernel.root_dir%/Resources/pages'
-                     type: page
-                 homepage:
-                     path: '%kernel.root_dir%/Resources/pages'
-                     type: home
-                 templates:
-                     path: '%kernel.root_dir%/Resources/snippets'
-                     type: snippet
```

Comment:
Where is that gone? Is that prepended now?

Reply:
it now use the default configuration `%kernel.root_dir%/config/templates/snippets` thats why the folder was also moved